### PR TITLE
Add more comments for DevicePluginHandlerImpl struct

### PR DESCRIPTION
**Pull request by ScorpioCPH**
**What this PR does / why we need it**:

Add more comments

**Special notes for your reviewer**:

@jiayingz PTAL.

**Release note**:

```
NONE
```

### CODE REVIEW

**Comment by ScorpioCPH** [comment]
@jiayingz PTAL :)

**Comment by tengqm** [comment]
Looks like the PR is adding little useful information while deleting some critical ones.

File: pkg/kubelet/cm/deviceplugin/device_plugin_handler.go [hunk 1]
```diff
@@ -61,12 +61,14 @@ type Handler interface {
 type HandlerImpl struct {
 	// TODO: consider to change this to RWMutex.
 	sync.Mutex
+	// devicePluginManager is an implementation of deviceplugin.Manager interface.
 	devicePluginManager Manager
 	// activePods is a method for listing active pods on the node
 	// so the amount of pluginResources requested by existing pods
 	// could be counted when updating allocated devices
 	activePods ActivePodsFunc
-	// devicePluginManagerMonitorCallback is used for testing only.
```
Comment (BenTheElder) [inline pkg/kubelet/cm/deviceplugin/device_plugin_handler.go]: if this is indeed used for testing only we should probably retain this..

Comment (ScorpioCPH) [reply pkg/kubelet/cm/deviceplugin/device_plugin_handler.go]: @BenTheElder thanks, I think this `Callback` is not for test now, @jiayingz WDYT?

**Pull request merged**